administrator/language/de-DE/com_users.ini

```
@@ -6,7 +6,7 @@
 
 COM_USERS="Benutzer"
 COM_USERS_A11Y_SETTINGS_FIELD_CONTRAST="Hoher Kontrast"
-COM_USERS_A11Y_SETTINGS_FIELD_FONTSIZE="Schriftgröße vergrößern"
+COM_USERS_A11Y_SETTINGS_FIELD_FONTSIZE="Schriftgröße erhöhen"
```

```suggestion
COM_USERS_A11Y_SETTINGS_FIELD_FONTSIZE="Schrift vergrößern"
```

ist das nicht klarer?

---

Ja finde auch dass das so klarer ist.

---

Ist typografisch so nicht ganz korrekt, aber damit kann ich leben